cmd/jaeger/config-elasticsearch.yaml

```diff
@@ -20,10 +20,35 @@ extensions:
     backends:
       some_storage:
         elasticsearch:
-          index_prefix: "jaeger-main"
```

Comment:
TODO: we may want to rethink if we still want a single field in the config for index prefix, which would apply if the individual index options didn't override it explicitly. This would simplify the configuration while maintaining the flexibility.

Reply:
@yurishkuro is this still something we want to address?

Reply:
I think it would be a good usability improvement - perhaps under Indices we can have a "defaults" entry that will apply unless overwritten by the more specific entries like `spans:`

Reply:
@yurishkuro this PR is already pretty big (31 files changed). thoughts on getting this PR in and tackling this in a follow-up PR under https://github.com/jaegertracing/jaeger/issues/6059. There's some follow-up work that needs to be done there anyway for aligning the config with OTEL.

Reply:
sure

Reply:
btw, it might be a good idea to try out [Sapling](https://sapling-scm.com/) and do stacked PRs, to avoid precisely this problem (at work we always use stacked PRs)